src/applications/appeals/995/tests/components/ITFBanner.unit.spec.jsx

```diff
@@ -41,4 +42,27 @@ describe('ITFBanner', () => {
       render(<ITFBanner status="nonsense" />);
     }).to.throw();
   });
+
+  it('should navigate back from error', () => {
+    const router = { push: sinon.spy() };
+    const { container } = render(<ITFBanner status="error" router={router} />);
+    fireEvent.click($('a', container)); // back
+  });
+  it('should navigate back from ITF found page', () => {
+    const router = { push: sinon.spy() };
+    const { container } = render(
+      <ITFBanner status="itf-found" router={router} />,
+    );
+    $('va-button-pair', container).__events.secondaryClick(); // back
+    expect(router.push.args[0][0]).to.contain('/introduction');
+  });
+  it('should dismiss ITF page', () => {
+    const { container } = render(
+      <ITFBanner status="itf-found">
+        <div id="done">hello</div>
```

Comment:
what's the `id=done` for?

Reply:
When the `ITFBanner` is dismissed, the content inside `<div id="done">...</div>` is rendered. We're checking that in the test below.